networking/SimpleTileEntities.md

```diff
@@ -0,0 +1,58 @@
+Simple Tile Entity Server to Client Networking
+=============================
+
+If you want to keep the client side tile entity synced with the server, you do not need the full networking system
+
+You can split the nbt saving/loading for the syncable items into their own functions
+
+    @Override
+    public void writeToNBT(NBTTagCompound tag)
+    {
+        super.writeToNBT(tag);
+        writeCustomNBT(tag);
+    }
+    
+    public void writeCustomNBT(tag)
+    {
+        tag.setString("active","yes");
+    }
+    
+    @Override
+    public void readFromNBT(NBTTagCompound tag)
+    {
+        super.writeToNBT(tag);
+        writeCustomNBT(tag);
+    }
+    
+    public void readCustomNBT(tag)
+    {
+        active=tag.setString("active");
+    }
+    
+Then override the getDescriptionPacket and onDataPacket
+
+    @Override
+    public Packet getDescriptionPacket()
+    {
+        NBTTagCompound nbttagcompound = new NBTTagCompound();
+        writeCustomNBT(nbttagcompound);
```

Comment:
Why NBT? Why the exact same values from read/write NBT?
No explanation of what actually should be synced and why.
You're just giving copy pasta code which is BAD.